arango/request.py

```diff
@@ -12,7 +12,7 @@ def normalize_headers(
     if driver_flags is not None:
         for flag in driver_flags:
             flags = flags + flag + ";"
-    driver_version = "7.7.0"
+    driver_version = "9.0.0"
```

Comment:
This is not related, but we need to update it before the upcoming release.

Reply:
Am I understanding correctly that this PR warrants another major release? 

Is there a way we could keep the driver to `8.x.x` for these changes?

Reply:
It's not particularly this PR , but this one: https://github.com/arangodb/python-arango/pull/192. 
But you're right, we could keep the driver to 8.x.x, it's not that big of a deal.